compiler/rustc_error_codes/src/error_codes/E0726.md

```diff
@@ -0,0 +1,47 @@
+When a struct or a type is bound/declared with a lifetime it is important for
```

Comment:
Please write a short summary of the error here, the explanation goes below. :)

Something like "An argument's lifetime was elided in an async function".